app/controllers/hyrax/my/collections_controller.rb

```diff
@@ -9,7 +9,7 @@ def index
         add_breadcrumb t(:'hyrax.controls.home'), root_path
         add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
         add_breadcrumb t(:'hyrax.admin.sidebar.collections'), hyrax.my_collections_path
-
+        list_presenter
```

Comment:
Prefer collection_type_list_presenter to avoid confusion with the list of collections that are on the admin index page.